lib/src/main/java/com/cloudbees/groovy/cps/impl/LocalVariableBlock.java

```diff
@@ -0,0 +1,58 @@
+package com.cloudbees.groovy.cps.impl;
+
+import com.cloudbees.groovy.cps.Continuation;
+import com.cloudbees.groovy.cps.Env;
+import com.cloudbees.groovy.cps.LValue;
+import com.cloudbees.groovy.cps.LValueBlock;
+import com.cloudbees.groovy.cps.Next;
+import org.codehaus.groovy.runtime.ScriptBytecodeAdapter;
+
+/**
+ * Access to local variables and method parameters.
+ *
+ * @author Kohsuke Kawaguchi
+ */
+public class LocalVariableBlock extends LValueBlock {
+    private final String name;
+    private SourceLocation loc;
+
+    public LocalVariableBlock(SourceLocation loc, String name) {
+        this.name = name;
+        this.loc = loc;
```

Comment:
SpotBugs noticed that `loc` was never actually assigned in this class, which seemed like a bug, so I added this line. Maybe it will improve the quality of some error messages? Or maybe `loc` was left `null` intentionally and I should suppress the error instead?